hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/container/ContainerInfo.java

```diff
@@ -219,7 +226,6 @@ public void updateLastUsedTime() {
   public HddsProtos.ContainerInfoProto getProtobuf() {
     HddsProtos.ContainerInfoProto.Builder builder =
         HddsProtos.ContainerInfoProto.newBuilder();
-    Preconditions.checkState(containerID > 0);
```

Comment:
Why drop this?

Reply:
Inside the ContainerID constructor, the same pre-condition is there:

```
  public ContainerID(long id) {
    Preconditions.checkState(id >= 0,
        "Container ID should be positive. %s.", id);
    this.id = id;
  }
```

So if we have a  ContainerID instance inside this class, which we now must have rather than the long value, it must have a value greater than zero making this check redundant.

Reply:
This is existing code inside the ContainerID class - I have not changed it. The fact that things worked OK with > 0 in the report processing code implies container ID must start at 1. I can change this if you like.